Makefile

```diff
@@ -0,0 +1,95 @@
+# It's necessary to set this because some environments don't link sh -> bash.
+SHELL := /bin/bash
+
+#-----------------------------------------------------------------------------
+# VERBOSE target
+#-----------------------------------------------------------------------------
+
+# When you run make VERBOSE=1 (the default), executed commands will be printed
+# before executed. If you run make VERBOSE=2 verbose flags are turned on and
+# quiet flags are turned off for various commands. Use V_FLAG in places where
+# you can toggle on/off verbosity using -v. Use Q_FLAG in places where you can
+# toggle on/off quiet mode using -q. Use S_FLAG where you want to toggle on/off
+# silence mode using -s...
+VERBOSE ?= 1
+Q = @
+Q_FLAG = -q
+QUIET_FLAG = --quiet
+V_FLAG =
+S_FLAG = -s
+X_FLAG =
+ifeq ($(VERBOSE),1)
+       Q =
+endif
+ifeq ($(VERBOSE),2)
+       Q =
+       Q_FLAG =
+       QUIET_FLAG =
+       S_FLAG =
+       V_FLAG = -v
+       X_FLAG = -x
+endif
+
+#----------------------------------------------------------------
+# HELP target
+#----------------------------------------------------------------
+
+## This target is used to display all other target
+.PHONY: help
+# Based on https://gist.github.com/rcmachado/af3db315e31383502660
+## Display this help text
+help:/
+	$(info Available targets)
+	$(info -----------------)
+	@awk '/^[a-zA-Z\-%\_0-9]+:/ { \
+		helpMessage = match(lastLine, /^## (.*)/); \
+		helpCommand = substr($$1, 0, index($$1, ":")-1); \
+		if (helpMessage) { \
+			helpMessage = substr(lastLine, RSTART + 3, RLENGTH); \
+			gsub(/##/, "\n                                     ", helpMessage); \
+			printf "%-35s - %s\n", helpCommand, helpMessage; \
+			lastLine = "" \
+		} \
+	} \
+	{ hasComment = match(lastLine, /^## (.*)/); \
+          if(hasComment) { \
+            lastLine=lastLine$$0; \
+	  } \
+          else { \
+	    lastLine = $$0 \
+          } \
+        }' $(MAKEFILE_LIST)
+
+
+
+# By default the project should be build under GOPATH/src/github.com/<orgname>/<reponame>
+GO_PACKAGE_ORG_NAME ?= $(shell basename $$(dirname $$PWD))
```

Comment:
This won't be required as we are using go modules.

Reply:
Its not standard go project path, if you look at it `goapth` is missing, we can add any path before it, it could be `gopath` or `dir_xyz` but for base or as default I took same path as pkg path in go. The variable GO_PACKAGE_PATH is currently not used anywhere but could be used for `Dockerfile` or for build location.

Reply:
@Avni-Sharma Have we decided that we are going to use gomod for service-binding operator ?

Reply:
@akashshinde Yes, it is Go modules.